Spec/RealtimeClientConnection.swift

```diff
@@ -2318,7 +2321,7 @@ class RealtimeClientConnection: QuickSpec {
                                     guard let errorInfo = errorInfo else {
                                         fail("ErrorInfo is nil"); done(); return
                                     }
-                                    expect(errorInfo.code).to(equal(40142)) //Token expired
+                                    expect(errorInfo.code).to(equal(-1003))
```

Comment:
Sorry, why -1003?  Where did that error code come from?  Here's our list of error codes, https://github.com/ably/ably-common/blob/master/protocol/errors.json